src/node_file.cc

```diff
@@ -882,6 +888,70 @@ static void MKDir(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+
+static size_t CountSlashes(const char* str, size_t len) {
+  size_t cntr = 0;
+  for (size_t i = 0; i < len; i++) {
+    if (str[i] == '/') cntr++;
```

Comment:
I couldn't see in any of the win calls of `uv_fs_realpath()` that could return `ERROR_CANT_RESOLVE_FILENAME` (mapped to `UV_ELOOP`). Specifically those are `CreateFileW()` and `pGetFinalPathNameByHandleW()`. I just added a couple tests that will push up in a moment and run CI to check.